source/_posts/2023-10-01-php-core-roundup-17.md

```diff
@@ -0,0 +1,122 @@
+
+---
+title: 'PHP Core Roundup #17'
+layout: post
+tags:
+  - roundup
+author:
+  - name: Ayesh Karunaratne
+    url: https://aye.sh
+  
+  - name: Sergey Panteleev
+    url: https://sergeypanteleev.com
+published_at: 01 October 2023
+
+---
+
+Welcome back to [PHP Core Roundup](/blog/tag/roundup/) series! This is post #17, where we highlight and celebrate the improvements made to PHP during the month past by the PHP development team, members of the PHP Foundation, and more.
+
+> The PHP Foundation is a collective of PHP contributors, veterans, and companies that collaborate to ensure the long-term sustainability of the PHP programming language. The foundation currently supports six contributors focused on PHP's maintenance, debt reduction, and feature development. These contributors work closely with others on coding, documentation, and discussions.
+
+> Started over a year ago, the PHP Core Roundup series offers a summary of the latest developments, discussions, and news about PHP Core, contributed by both PHP Foundation members and other participants. This post is the thirteenth in the PHP Core Roundup series.
+
+{% include "newsletter.html" %}
+
+## Early-developments for PHP 8.4
+
+Although PHP 8.3 is still being ironed out, there are some discussions and even an RFC currently being voted for proposed changes in PHP 8.4 (scheduled for the end of 2024).
+
+[Deprecations for PHP 8.4 RFC](https://wiki.php.net/rfc/deprecations_php_8_4) is currently in draft, which stands to track ideas on deprecating certain features. So far those are related to the `DOMAttr::schemaTypeInfo()`, `DOMElement::schemaTypeInfo()`, `DOMImplementation::getFeature()`, `mysqli_ping()`, `mysqli::ping()` functions, and `DOM_PHP_ERR` constant.
```

Comment:
It feels odd that this one is specifically mentioned at the top, despite being a draft, whereas the BCrypt cost one (which is in voting for PHP 8.4) is mentioned further down.

Reply:
Agreed.
Also schemaTypeInfo is a property not a method, so doesn't need () at the end.